conference-finder/src/ConferenceItem.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const ConferenceItem = ({ conference }) => {
```

Comment:
You could destructure conference here so you won't have to type conference.x, conference.y, conference.z over and over again :) Also, since you're only returning some JSX here without some more logic you could simply return without the curly braces.

So the method declaration would look like this:

const ConferenceItem = ({conferenceName, conferenceTopic, conferenceDateTime, city}) => 
`[ignore the backticks, it's just so github won't render this as html](
        <li>
            <div class="card">
                <h3>
                    {conferenceName}
                </h3>
                <p>
                    {conferenceTopic}
                    {conferenceDateTime}
                    {city}
                </p>
            </div>
        </li>
    )`

Reply:
Alternatively, pass in the full object as a prop then destructure before the return. 
`const {conferenceName, conferenceTopic, conferenceDateTime, city} = conference;`

Reply:
Though in the syntax above you'd still pass in only one object, too. Your solution might be a little bit more readable though :)

Reply:
We've now got it passing the full object in and restructuring it on the next line.